js/feed.js

```diff
@@ -0,0 +1,301 @@
+import paramenterManagement from "../utils/utils.mjs";
+import postManagement from "../modules/posts.mjs";
+
+const API_BASE_URL = "https://api.noroff.dev/api/v1";
```

Comment:
I can see you repeating this in two files. You can extract this to a constant or utils file and import it to other files.